src/credentials/CHIPCert.cpp

```diff
@@ -318,6 +318,17 @@ CHIP_ERROR ChipCertificateSet::LoadCerts(TLVReader & reader, BitFlags<CertDecode
     return err;
 }
 
+CHIP_ERROR ChipCertificateSet::ReleaseLastCert()
+{
+    const ChipCertificateData * lastCert = GetLastCert();
+    VerifyOrReturnError(lastCert != nullptr, CHIP_ERROR_INTERNAL);
+
+    lastCert->~ChipCertificateData();
```

Comment:
This is very odd: a const pointer, but were we call what is ostensibly a mutating destructor. Is there a cleaner way to do this, or a new public method that can be added to ChipCertificateData? Does this leak anything?